src/docs/product/performance/retention-priorities/index.mdx

```diff
@@ -0,0 +1,127 @@
+
+Below is a list of the transactions we prioritize and deprioritize by default. Most can be customized on a per-project basis.
+
+### Prioritized Transactions
+
+Sentry automatically prioritizes and retains the following types of transactions:
+
+#### Latest Release
+
+Latest releases are a retention priority because when you create a new release, we assume you'll want to have more visibility during the early adoption phase which improves your ability to catch new issues as your release is gaining traction.
+Because you may want to deploy the same release in different environments, we'll also take your environment into account and prioritize those transactions.
+
+#### Dev Environments
+
+Since dev environments usually generate a small number of transactions as compared to prod environments, we prioritize them during your testing phase so you can get meaningful insights.
+
+#### Low Volume Projects
+
+<Note>
+
+You won't be able to disable the Low Volume Projects retention priority because it's set on an organization-wide level not on a per-project level basis.
+
+</Note>
+
+Some projects within your organization are very high volume compared to others. This means that low-volume projects might get drowned out if we treat all projects equally. To ensure that you have enough samples regardless of project scale, we prioritize low-volume projects.
+
+#### Low Volume Transactions
+
+Transactions can be logically grouped by their name within a project. This means that transactions with the same name most likely indicate the same operation or code path, while transactions with different names usually indicate different operations.
+We prioritize low volume transactions for the same reason we prioritize low volume projects - so they don't get drowned out by high volume transactions. We want to make sure that transactions representing low volume code paths are sampled in enough quantities to generate a complete view of your application.
+
+### Deprioritized Transactions
+
+Sentry automatically deprioritizes the following transactions:
+
+#### Health Checks
+
+We deprioritize health check type transactions because while they're important for checking the stability of your application, they don't have any value beyond the task associated with them. We consider a transaction to be a health check if its name matches one of the following glob patterns:
+
+- `*healthcheck*`
+- `*healthy*`
+- `*live*`
+- `*ready*`
+- `*heartbeat*`
+- `*/health`
+- `*/healthz`
+- `*/ping`
+
+### How to Change Retention Priorities
+
+To make changes to a project's retention priorities in Sentry, go to **Settings > Projects**, select the project you're interested in, then scroll down to "Performance" in the "PROCESSING" section and make any updates under "RETENTION PRIORITIES" using the toggles. Each of your projects can have an individual set of retention priorities.
+
+<div style="position: relative; padding-bottom: calc(52.60977630488816% + 41px); height: 0; width: 100%">
```

Comment:
I love the arcade demo :D I just had trouble on the "scroll down to Performance" step, because when I actually try to click on Performance the bottom navigation things pops up... It's really minor, since people can just go to the next step, but if the page were a bit scrolled up this would work as well.

Reply:
I know, it drives me crazy! I tried it a bunch of different ways, but I couldn't get the left nav in our product to scroll so that 'performance' appeared higher up and didn't have that issue :(